homeassistant/components/mqtt/binary_sensor.py

```diff
@@ -221,7 +221,7 @@ def state_message_received(msg: ReceiveMessage) -> None:
                     self._config.get(CONF_VALUE_TEMPLATE),
                 )
                 return
-
+            monitor = EntityMonitor(self, {"_attr_is_on"})
```

Comment:
Maybe bind this to the entity at creation time so its not created every time

Reply:
On the other hand I want to remove the monitored attrs after processing